### ISSUE TITLE
Iris create with MV, Invalid Generator: Iris

### ISSUE THREAD

**Issue by cyberpwnn** [VolmitSoftware/Iris#234]
Multiverse doesnt know who we are, no idea why. Logs would be helpful

**Comment by FluffyDiscord** [comment]
https://hastebin.com/etixudiver.md

**Comment by cyberpwnn** [comment]
Yeah multiverse doesnt know where iris is? Odd.... 

```
[21:58:17 INFO]: Mossshine issued server command: /iris create ljahkjghdajkshgkljfdhgld
[21:58:17 INFO]: [Iris]: test is configured to generate Overworld!
[21:58:18 INFO]: [Iris]: Injecting 100 biomes into the NMS World Chunk Provider (Iris)
[21:58:18 INFO]: Preparing start region for dimens... Oh wait, We don't do that here anymore.
[21:58:18 WARN]: [Iris] Task #2227 for Iris v1.2.4-hotfix1 generated an exception
java.lang.NullPointerException: null
        at com.volmit.iris.manager.command.CommandIrisCreate.lambda$null$2(CommandIrisCreate.java:173) ~[?:?]
        at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftTask.run(CraftTask.java:99) ~[patched_1.16.4.jar:git-Paper-389]
        at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:468) ~[patched_1.16.4.jar:git-Paper-389]
        at net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1289) ~[patched_1.16.4.jar:git-Paper-389]
        at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:377) ~[patched_1.16.4.jar:git-Paper-389]
        at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1204) ~[patched_1.16.4.jar:git-Paper-389]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:992) ~[patched_1.16.4.jar:git-Paper-389]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:173) ~[patched_1.16.4.jar:git-Paper-389]
        at java.lang.Thread.run(Unknown Source) [?:?]
```

That error is only happening because iris expected mv to create a world but it diddnt. So iris failed also.

**Comment by FluffyDiscord** [comment]
`/mvgens` which should show Iris as an generator does not show Iris. This did work back with Iris version 1.1.10

**Comment by FluffyDiscord** [comment]
Absolute clean setup - paper#389, no other folders than `plugins` with iris and MV, the `iris` error is back from VolmitSoftware/IrisPlugin#280: https://hastebin.com/ikocekoyer.md

That error is related to `overworld` pack, missing/not downloading.

After manual download with `/iris download overworld` and running `/iris create test` the world is NOT created. Iris is still not registered in MV as a generator and throws silent error. https://hastebin.com/lopejulixu.yaml
![image](https://user-images.githubusercontent.com/10332247/103700196-6bfe7080-4fa4-11eb-89cd-a2fa88debd0d.png)

**Comment by cyberpwnn** [comment]
Fixed in 1.2.5
